src/Combinatorics/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,211 @@
+export chow_ring, augmented_chow_ring, select
+
+@doc Markdown.doc"""
+The Chow ring of a matroid, optional also with the simplical generators.
```

Comment:
By general convention, Julia docstrings should always start with the function signature, indented by four spaces (thus turning it into a codeblock).
```suggestion
    chow_ring(M::Matroid; extended=false)

Return the Chow ring of a matroid, optional also with the simplicial generators.
```